test/unit/Reflection/ReflectionParameterTest.php

```diff
@@ -465,7 +467,7 @@ public function testIsDefaultValueConstantAndGetDefaultValueConstantName() : voi
         $method = $classInfo->getMethod('methodWithConstAsDefault');
         $constDefault = $method->getParameter('constDefault');
         self::assertTrue($constDefault->isDefaultValueConstant());
-        self::assertSame('SOME_CONST', $constDefault->getDefaultValueConstantName());
+        self::assertSame('self::SOME_CONST', $constDefault->getDefaultValueConstantName());
```

Comment:
This should be resolved to a FQCN, not to `self`. `self` is an alias with no meaning when put out of context.